src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -86,7 +86,13 @@ export const validationSchema = Yup.object()
     logoImageFile: createFileSchema("logoImageFile", false).notRequired(),
 
     showGrid: Yup.bool().notRequired(),
-    columns: Yup.number().notRequired().min(1).max(100),
+    columns: Yup.number().when("showGrid", {
+      is: true,
+      then: Yup.number()
+        .required("The columns need to be between 5 and 100.")
+        .min(5)
+        .max(100),
+    }),
```

Comment:
As per other comment, using the shared `const`s for these would be good.

Reply:
This raises another question too, we seem to have similar (if not the same) validation rules shared between this file and `src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx`. Are they the same? And if so, can we reduce the duplication of them and use the same definition for both?